### DISCUSSION TITLE
Tuning x509 in Files Analyzer

### DISCUSSION THREAD

**Discussion by WDV71673** [Security-Onion-Solutions/securityonion#7697] [General] [answered]
Is there a way to tune out the x509 analyzer from the Zeek files.log? We are receiving an overwhelming amount of x-x509-ca-cert and x-x509-user-cert mimetypes that we would like to tune out.

**Comment by dougburks** [comment, marked as answer]
If you want to stop ingesting Zeek x509 logs altogether, you can use `so-zeek-logs`:
https://docs.securityonion.net/en/2.3/so-zeek-logs.html
